live-build/misc/ansible-roles/appliance-build.install-delphix-s3-packages/tasks/main.yml

```diff
@@ -42,7 +42,8 @@
   register: tempdir
   delegate_to: localhost
 
-- command: aws s3 sync --only-show-errors "s3://{{ bucket }}/{{ prefix }}" .
+- name: Download packages
```

Comment:
IMO, I'd leave off the name, to be consistent with the other tasks..

I'd like to make a change that adds the name to all the tasks, and then enables the ansible lint check to ensure we always name tasks moving forward.. but until all other tasks are updated, I vote to be consistent, and not add a name here (and below).